ephios/event_management/forms.py

```diff
@@ -46,6 +47,41 @@ class Meta:
         model = Event
         fields = ["title", "description", "location", "type", "mail_updates"]
 
+    def __init__(self, **kwargs):
+        user = kwargs.pop("user")
+        can_publish_for_groups = get_objects_for_user(user, "publish_event_for_group", klass=Group)
+
+        if (event := kwargs.get("instance", None)) is not None:
+            responsible_persons = get_users_with_perms(
```

Comment:
should rename to responsible_users here as well

Reply:
the own user that is prefilled in initial still does not get saved/assigned change permissions. not sure if this is related to this renaming issue

Reply:
it probably was